### PR TITLE
Refactor frame types

### PR DESCRIPTION
**Pull request by VLitanADI**

### CODE REVIEW

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 1]
```diff
@@ -326,16 +317,37 @@ aditof::Status CameraItof::getAvailableFrameTypes(
     return status;
 }
 
+aditof::Status setAttributesByMode(aditof::Frame* frame, const ModeInfo::modeInfo& modeInfo){
+    aditof::Status status = aditof::Status::OK;
+//assume frame is != nullptr as it is checked before calling this
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:322]: Why not pass the frame by reference to ensure is not null?

File: sdk/src/cameras/itof-camera/dragonboard/sensor_enumerator_dragonboard.cpp [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/*
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/dragonboard/sensor_enumerator_dragonboard.cpp:1]: there shouldn't be a 'dragonboard' folder

File: sdk/src/cameras/itof-camera/camera_itof.cpp (hunk 1 above)
Comment (VLitanADI) [reply sdk/src/cameras/itof-camera/camera_itof.cpp]: done

File: sdk/src/cameras/itof-camera/dragonboard/sensor_enumerator_dragonboard.cpp (hunk 2 above)
Comment (VLitanADI) [reply sdk/src/cameras/itof-camera/dragonboard/sensor_enumerator_dragonboard.cpp]: fixed

**Review by dNechita — APPROVED** [review]

**Pull request merged**